binding/jaxrs2/README.md

```diff
@@ -25,29 +25,31 @@ Please add the following dependencies to enable TracEE JAX-WS support. For examp
 If you don't use an explicit application config and have the `tracee-jaxrs2` module on your classpath, 
 all filters will be automatically applied to your JAX-RS services.
 
-If you want to use an explicit `Application` class, you have to `TraceeContainerRequestFilter`
+If you use an explicit `Application` class, you need to add `TraceeContainerRequestFilter`
 and `TraceeContainerResponseFilter` as provider classes.
 
+Example:
+
 ```java
-@ApplicationPath("/basepath")
+@ApplicationPath("/myApplication")
 public class ApplicationConfig extends Application {
 
     public Set<Class<?>> getClasses() {
-        return new HashSet<Class<?>>(Arrays.asList(io.tracee.jaxrs.container.TraceeContainerRequestFilter.class,
-        io.tracee.jaxrs.container.TraceeContainerResponseFilter.class, ...);
+        return new HashSet<Class<?>>(Arrays.asList(io.tracee.binding.ws.rs.TraceeContainerRequestFilter.class,
+        io.tracee.binding.ws.rs.container.TraceeContainerResponseFilter.class, ...);
```

Comment:
Why do you move the entire JAX-RS2-Stack to a cumbersome package `binding.ws.rs`? I prefer a namescheme similar to jaxws: ìo.tracee.binding.jaxrs`.